src/utils/runWithTruffle.js

```
@@ -19,6 +23,17 @@ function initTruffle(network) {
   });
 }
 
-export default function runWithTruffle(script, network) {
+function compileWithTruffle() {
+  log.info("Compiling contracts with truffle")
```

I think truffle already outputs this one for you

---

I left that one in case truffle fails silently.